ui/src/App.tsx

```diff
@@ -172,7 +160,7 @@ const AppComponent = () => {
     sidebar: [
       {
         label: 'Auctions',
-        path: paths.app.distribution.auctions,
+        path: paths.app.auctions.root,
```

Comment:
auctions are currently not nested in the distribution tab, however maybe they should be in which case we can nest the auction pages inside distribution